relay/cache/cache_accessor.go

```diff
@@ -2,9 +2,11 @@ package cache
 
 import (
 	"context"
-	"golang.org/x/sync/semaphore"
 	"sync"
 	"time"
+
+	cache2 "github.com/Layr-Labs/eigenda/common/cache"
```

Comment:
Consider calling this `cachecommon` (which you've used elsewhere) instead of `cache2` (comment applies to other files where you used `cache2` as well)

Reply:
done